test_e2e.sh

```diff
@@ -1072,6 +1074,81 @@ function e2e::exechook_fail_retry() {
     fi
 }
 
+##############################################
+# Test exechook-success with GIT_SYNC_ONE_TIME
+##############################################
+function exechook_success_once_helper() {
+    # First sync
+    echo "$FUNCNAME 2" > "$REPO"/file
+    git -C "$REPO" commit -qam "$FUNCNAME 2"
+
+    GIT_SYNC \
+        --period=${2}ms \
+        --one-time \
+        --repo="file://$REPO" \
+        --branch="$MAIN_BRANCH" \
+        --root="$ROOT" \
+        --link="link" \
+        --exechook-command="$EXECHOOK_COMMAND_SLEEPY" \
+        >> "$1" 2>&1 &
+    sleep 7
```

Comment:
can we reduce sleeps to minimum?

Reply:
You mean reducing total time test components spend sleeping correct? I need to sleep in order to add extra time for the hook to complete, to ensure that main thread is waiting for hook to complete before terminating.